components/layout/FormLayout.module.css

```diff
@@ -11,14 +11,22 @@
 
 .row {
   display: flex;
+  flex-wrap: wrap;
   align-items: center;
   position: relative;
   margin-bottom: 20px;
   line-height: 1.8;
+  padding: 0 6px;
```

Comment:
This is just personal preference but I'd increase the padding to something around `20px`

Before - After:
<img height="500" alt="Screenshot 2021-03-08 at 09 31 51" src="https://user-images.githubusercontent.com/24435169/110295202-20dafb00-7ff1-11eb-9811-dd7f31534013.png"> <img height="500" alt="Screenshot 2021-03-08 at 09 23 35" src="https://user-images.githubusercontent.com/24435169/110295157-115bb200-7ff1-11eb-8fa9-c4c85abab227.png">

Reply:
Thanks. Yeah, that would be fine with me. My concern was just to have it fit on the screen. Before changing it though, I'd like to know if @mikecao also agrees with increasing the padding or has any additional concerns.

Reply:
Yea, a little more padding would be nice.

Reply:
No problem. I've adjusted it.